pkg/volume/azure_dd/azure_dd.go

```diff
@@ -188,7 +188,7 @@ func getMaxDataDiskCount(instanceType string, sizeList *[]compute.VirtualMachine
 			continue
 		}
 		if strings.ToUpper(*size.Name) == vmsize {
-			glog.V(2).Infof("got a matching size in getMaxDataDiskCount, Name: %s, MaxDataDiskCount: %s", *size.Name, *size.MaxDataDiskCount)
+			glog.V(2).Infof("got a matching size in getMaxDataDiskCount, Name: %v, MaxDataDiskCount: %v", *size.Name, *size.MaxDataDiskCount)
```

Comment:
wrong datatype on the %'s - they're numeric not string

Reply:
We should keep the first `%s`, *size.Name is string.

Reply:
`%v` works just fine for strings, but I'll switch it back to `%s`.
I don't think it was a linter error, is linter run as part of "make test" ?
The console showed this:
```
# k8s.io/kubernetes/cmd/kubeadm/app/phases/certs
cmd/kubeadm/app/phases/certs/certs.go:113: Verbose.Infoln call has possible formatting directive %s
```

Reply:
> is linter run as part of "make test"

i don't think so.